scripts/build-docs.js

```diff
@@ -47,6 +47,8 @@ const generateReference = async () => {
         if (!output[nameParts[0]][nameParts[1]][nameParts[2]]) {
           output[nameParts[0]][nameParts[1]][nameParts[2]] = {};
         }
+      } else if (nameParts.length > 3){
+        console.log("Error: Not all documentation is being generated. Missing: ", nameParts.join('.'));
```

Comment:
Added this check just in case we ever get very deeply nested documentation. It should not be called and if it is, we would need to update this code.

Reply:
Great